Makefile

```diff
@@ -4,15 +4,22 @@ CRDS := containerjfr flightrecorder
 .DEFAULT_GOAL := bundle
 
 .PHONY: generate
-generate: k8s openapi
+generate: openapi k8s crds
 
 .PHONY: k8s
 k8s:
 	operator-sdk generate k8s
 
+.PHONY: crds
+crds:
+	operator-sdk generate crds
+
 .PHONY: openapi
-openapi:
-	operator-sdk generate openapi
+openapi: bin/openapi-gen
+	./bin/openapi-gen --logtostderr=true -o "" -i ./pkg/apis/rhjmc/v1alpha1 -O zz_generated.openapi -p ./pkg/apis/rhjmc/v1alpha1 -h ./boilerplate-header.txt -r "-"
+
+bin/openapi-gen:
+	go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
Do we know if we need `zz_generated.openapi.go`? According to https://github.com/operator-framework/operator-sdk/pull/2276, `generate crds` does everything `generate openapi` used to do except generate this file. It also says that the file is not needed for most use cases. If we don't need them, perhaps we should just delete those files.

Reply:
It doesn't seem like we do, so I've just pushed a commit to remove it. If we do need it back then it's easy enough to get.